docker/muranoservice/README.md

```diff
@@ -0,0 +1,237 @@
+# Installing murano as service
+## Install murano packages
+
+Enable Openstack repositories:
+
+    apt install software-properties-common
+    add-apt-repository cloud-archive:newton
+
+Then, install murano packages:
+
+    apt-get install murano-api murano-engine python-muranoclient
+
+## Create database
+
+To create the database, complete these steps:
+
+Use the database access client to connect to the database server as the root user:
+
+    $ mysql -u root -p
+
+Create the murano database:
+
+    CREATE DATABASE murano;
+
+Grant proper access to the heat database:
+
+    GRANT ALL PRIVILEGES ON murano.* TO 'murano'@'localhost' \
+    IDENTIFIED BY 'MURANO_DBPASS';
+
+    GRANT ALL PRIVILEGES ON murano.* TO 'murano'@'%' \
+    IDENTIFIED BY 'MURANO_DBPASS';
+
+Replace MURANO_DBPASS with a suitable password.
+
+Exit the database access client.
+
+## Create endpoints
+
+Create the application-catalog service API endpoints:
+
+    $ keystone service-create --name application-catalog --type application-catalog \
+     --description "application-catalog"
+
+    $ keystone endpoint-create \
+     --service-id $(keystone service-list | awk '/ application-catalog / {print $2}') \
+     --publicurl http://controller:8082/v1/%\(tenant_id\)s \
+     --internalurl http://controller:8082/v1/%\(tenant_id\)s \
+     --adminurl http://controller:8082/v1/%\(tenant_id\)s \
+     --region regionOne
+
+## Configure murano
+Edit the /etc/murano/murano.conf file and complete the following actions:
+
+In the [database] section, configure database access:
```

Comment:
In the [database] section, configure database access, replacing MURANO_DBPASS with the password you chose for the Murano database: